domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -1711,10 +1738,19 @@ public TotaledResults(RowSet<Row> set, Integer total) {
     Handler<AsyncResult<PostgresClientStreamResult<T>>> replyHandler) {
 
     streamGet(table, clazz, fieldName, filter, returnIdField, distinctOn,
-      null, replyHandler);
+      null, replyHandler, 0);
   }
 
-  /**
+  public <T> void streamGet(String table, Class<T> clazz, String fieldName,
```

Comment:
Can you add javadoc for the parameters?
Can you add javadoc what unit (hours, minutes, ...) the timeout has?